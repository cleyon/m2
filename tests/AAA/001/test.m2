@define greet Hello, $1!  m2 sends you $0ings.
@greet world@
