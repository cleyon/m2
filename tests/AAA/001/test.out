Hello, world!  m2 sends you greetings.
